Azure/sonic-buildimage
# Add Cameo platform ESC601-32Q

Switch Vendor: Cameo
Switch SKU: ESC601-32Q
CPU: BROADWELL-DE
ASIC Vendor: Innovium
Switch ASIC: TL7
Port Configuration: 32x200G

    What I did
    Add a new Cameo platform ESC601-32Q.

    How I did it
    Add files by following SONiC Porting Guide.

    How to verify it
    a)with "make target/sonic-innovium.bin"
    b)install image and check the following commands
    decode-syseeprom
    sensors
    psuutil
    sfputil

**- Which release branch to backport (provide reason below if seleted)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
